rosidl_generator_py/resource/_msg.py.template

```diff
@@ -1,10 +1,30 @@
 # generated from rosidl_generator_py/resource/_msg.py.template
 # generated code does not contain a copyright notice
 
+import rclpy
+rclpy_implementation = rclpy._rclpy.rclpy_get_implementation_identifier()
+
+if rclpy_implementation == 'opensplice_static':
+    from @(spec.base_type.pkg_name).@(subfolder)._@(module_name)_support__rosidl_typesupport_opensplice_c import convert_from_py
+    from @(spec.base_type.pkg_name).@(subfolder)._@(module_name)_support__rosidl_typesupport_opensplice_c import convert_to_py
+    from @(spec.base_type.pkg_name).@(subfolder)._@(module_name)_support__rosidl_typesupport_opensplice_c import type_support
+elif rclpy_implementation == 'connext_static':
+    from @(spec.base_type.pkg_name).@(subfolder)._@(module_name)_support__rosidl_typesupport_connext_c import convert_from_py
+    from @(spec.base_type.pkg_name).@(subfolder)._@(module_name)_support__rosidl_typesupport_connext_c import convert_to_py
+    from @(spec.base_type.pkg_name).@(subfolder)._@(module_name)_support__rosidl_typesupport_connext_c import type_support
+else:
+    raise RuntimeError('Unknown implementation: {0}'.format(rclpy_implementation))
+
 
 class Metaclass(type):
     """Metaclass of message '@(spec.base_type.type)'."""
 
+    _CONVERT_FROM_PY = convert_from_py
+
+    _CONVERT_TO_PY = convert_to_py
+
+    _TYPE_SUPPORT = type_support
```

Comment:
Since it is not obvious what these variables are used for it would be good to add a comment.